cryptid_anchor/Anchor.toml

```diff
@@ -23,7 +25,11 @@ wallet = "~/.config/solana/id.json"
 
 [[test.genesis]]
 address = "didso1Dpqpm4CsiCjzP766BGY89CAdD6ZBL68cRhFPc"
-program = "../programs/cryptid_signer/tests/fixtures/sol_did_2.0.0.so"
+program = "tests/fixtures/sol_did_2.0.0.so"
+
+[[test.genesis]]
```

Comment:
Here we should have a quick discussion again if these programs should be test-suite specific or global (as it is right now). I would generally put the complete program config into a Test.toml and not the Anchor.toml. But let's discuss after merge.